src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Stepper/StepperHandler.Standard.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Xamarin.Platform.Handlers
+{
+	public partial class StepperHandler : AbstractViewHandler<IStepper, object>
+	{
+		protected override object CreateView() => throw new NotImplementedException();
```

Comment:
shouldn't that throw in the default (base) implementation ?

Reply:
But it is an abstract method.